ivy/functional/backends/jax/searching.py

```diff
@@ -30,7 +31,7 @@ def argmin(
 argmin.support_native_out = True
 
 
-def nonzero(x: JaxArray, *, out: Optional[JaxArray] = None) -> JaxArray:
+def nonzero(x: JaxArray, *, out: Optional[JaxArray] = None) -> Tuple[JaxArray]:
```

Comment:
we should not have the `out` argument for this function. The `out` argument is only relevant for functions which return a single array.